invenio_sequencegenerator/ext.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Flask extension for Invenio-SequenceGenerator."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint
+
+
+class InvenioSequenceGenerator(object):
+    """Invenio-SequenceGenerator extension."""
+
+    def __init__(self, app=None):
+        """Extension initialization."""
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        """Flask application initialization."""
+        self.init_config(app)
+        app.extensions['invenio-sequencegenerator'] = self
+
+        # Register dummy blueprint just for loading templates
+        template_loader = Blueprint('template_loader', __name__,
```

Comment:
- `invenio_sequencegenerator` instead of `template_loader`
- you can do it in one go:
```python
app.register_blueprint(Blueprint(
    'invenio_sequencegenerator', ...
)))
```